src/openassetio-core/include/openassetio/managerApi/ManagerInterface.hpp

```diff
@@ -467,9 +468,12 @@ class OPENASSETIO_CORE_EXPORT ManagerInterface {
   /**
    * Prepares for interaction with a host.
    *
-   * This is a good opportunity to initialize any persistent connections
-   * to a back end implementation. It is fine for this call to block for
-   * a period of time.
+   * This method is passed a settings dictionary, that can be used to
+   * configure required local state to service requests. For example,
```

Comment:
[consider][minor]?
```suggestion
   * configure required local state for service requests. For example,
```

Reply:
Was thinking of service as a verb - does that still work?

Reply:
Aye it does.